apps/meteor/tests/unit/app/settings/server/functions/settings.tests.ts

```diff
@@ -3,20 +3,32 @@ import { beforeEach, describe, it } from 'mocha';
 
 import { CachedSettings } from '../../../../../../app/settings/server/CachedSettings';
 import { SettingsRegistry } from '../../../../../../app/settings/server/SettingsRegistry';
+import { getSettingDefaults } from '../../../../../../app/settings/server/functions/getSettingDefaults';
 import { Settings } from '../../../../../../app/settings/server/functions/settings.mocks';
 
+const testSetting = getSettingDefaults({
+	_id: 'my_dummy_setting',
+	type: 'string',
+	value: 'dummy',
+});
+
+const settings = new CachedSettings();
+settings.initialized();
+
+Settings.settings = settings;
+
+const settingsRegistry = new SettingsRegistry({ store: settings, model: Settings as any });
+
 describe('Settings', () => {
 	beforeEach(() => {
+		settings.store.clear();
 		Settings.insertCalls = 0;
 		Settings.upsertCalls = 0;
+		Settings.data.clear();
 		process.env = {};
 	});
 
 	it('should not insert the same setting twice', async () => {
-		const settings = new CachedSettings();
-		Settings.settings = settings;
-		settings.initialized();
-		const settingsRegistry = new SettingsRegistry({ store: settings, model: Settings as any });
```

Comment:
Let's keep each test with its instance? to ensure it does exactly what it's supposed to do without interference? you can also test the clear cases, but as a specific case